wlutil/wlutil.py

```diff
@@ -19,6 +19,7 @@
 import re
 import pprint
 import doit
+import importlib
```

Comment:
Running `marshal build boards/firechip/base-workloads/br-base.json` on Python 3.6.8 throws an error:
```
Traceback (most recent call last):
  File "./marshal", line 218, in <module>
    main()
  File "./marshal", line 73, in main
    wlutil.initialize()
  File "/home/centos/chipyard/software/firemarshal/wlutil/wlutil.py", line 390, in initialize
    ctx = marshalCtx()
  File "/home/centos/chipyard/software/firemarshal/wlutil/wlutil.py", line 242, in __init__
    self.deriveOpts()
  File "/home/centos/chipyard/software/firemarshal/wlutil/wlutil.py", line 332, in deriveOpts
    m = self.importDistro(dPath)
  File "/home/centos/chipyard/software/firemarshal/wlutil/wlutil.py", line 293, in importDistro
    spec = importlib.util.spec_from_file_location(distroPath.stem, distroPath / "__init__.py",
AttributeError: module 'importlib' has no attribute 'util'
```
This happens even with a fresh clone and may be related to this [partial initialization issue](https://stackoverflow.com/questions/39660934/error-when-using-importlib-util-to-check-for-library).

I am able to work around it with
```suggestion
import importlib.util
```

Reply:
Oddly enough, the same import was used before without any problems.
https://github.com/firesim/FireMarshal/blob/5848bd3946806f2565165d23058d08b074b430de/wlutil/install.py#L5

Reply:
ya, Abe just found that one to. It's weird because it works fine on my machine. I think this is an issue with a specific version of importlib (I'm using anaconda, but still python 3.6). Anyway, your suggestion will fix it.